addon-resizer/README.md

```diff
@@ -91,6 +91,10 @@ spec:
           command:
             - /pod_nanny
             - --config-dir=/etc/config
+            - --cpu=300m
```

Comment:
So... heapster will be using 300m + 20m per node now? Also, why is nanny using 300m/200Mi?